clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -10969,6 +10969,12 @@ def warn_boolean_attribute_argument_is_not_valid: Warning<
 def err_sycl_attibute_cannot_be_applied_here
     : Error<"%0 attribute cannot be applied to a "
             "static function or function in an anonymous namespace">;
+def warn_sycl_pass_by_value_deprecated 
+    : Warning<"Passing kernel functions by value is deprecated in SYCL 2020.">,
```

Comment:
Nit: no periods at the end of these warnings.

Reply:
done